web/sites/default/settings.php

```diff
@@ -805,3 +805,14 @@
 $config_directories['sync'] = '../config/default';
 
 $settings['node_title_length_chars'] = 1023;
+$databases['default']['default'] = array (
```

Comment:
Rather copy this file into `web/sites/default/settings.local.php` and make these changes there.

Reply:
Done

Reply:
Done